### PR TITLE
adding python examples to treat yarp images

### PR DESCRIPTION
**Pull request by vtikha**
These are python examples for python2 and python3 on how to deal with a stream of yarp images

### CODE REVIEW

**Comment by drdanz** [comment]
The copyright check is failing. Please fix the copyright headers and/or the permission on the files (.py script should be executable).

**Comment by Nicogene** [comment]
> Please fix the copyright headers and/or the permission on the files

The copyright should be: 
```python
# Copyright (C) 2006-2019 Istituto Italiano di Tecnologia (IIT)
# All rights reserved.
#
# This software may be modified and distributed under the terms of the
# BSD-3-Clause license. See the accompanying LICENSE file for details
```

File: bindings/python/examples/yarpImage_py2.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/usr/local/bin python
```
Comment (drdanz) [inline bindings/python/examples/yarpImage_py2.py]:
```suggestion
#!/usr/bin/python
```

File: bindings/python/examples/yarpImage_py3.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
+#!/usr/local/bin python
```
Comment (drdanz) [inline bindings/python/examples/yarpImage_py3.py]:
```suggestion
#!/usr/bin/python3
```

**Comment by vtikha** [comment]
Addressed all the requested modifications.

**Comment by drdanz** [comment]
Rebased, squashed commits, and fixed the license (a `.` is missing in @Nicogene comment)

Please use a personal fork instead of yarp main repository next time, since this starts 2 build instead of one.

**Comment by drdanz** [comment]
Merged, thanks.

**Pull request merged**